src/Classes/Particle/Updater.ts

```diff
@@ -67,7 +87,28 @@ export class Updater {
 
                 const ctx = container.canvas.context;
 
-                const colorLine = container.particles.lineLinkedColor;
+                let colorLine: IRgb | undefined;
+
+                /*
+                 * particles connecting line color:
+                 *
+                 *  random: in blink mode : in every frame refresh the color would change hence resulting blinking of lines
+                 *  mid: in consent mode: sample particles color and get a mid level color from those two for the connecting line color
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)